content/learning-paths/cross-platform/vectorization-friendly-data-layout/a-more-complex-problem-revisited-sve.md

```diff
@@ -6,7 +6,7 @@ weight: 7
 layout: learningpathall
 ---
 
-You now know the logic behind the vectorization, so you should be confident that you can understand the SVE code shown below:
+You now know the logic behind the vectorization and should threfore be confident that you can understand the SVE code shown below:
```

Comment:
typo, therefore

Reply:
Thanks @markos, correction committed

Reply:
Apparently, I was too quick, there are 2 spellings, therefore and therefor (which is archaic and does not have the same meaning. Just use 'therefore' :)